client/stories/LinkTo.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { Button } from 'react-bootstrap'
```

Comment:
Please use an actual ANET component (from `src/components`) in the example. That way we know the whole storybook setup works with our own components.